src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarBattleSessionUtil.java

```diff
@@ -52,32 +52,31 @@ public static void evaluateBattleSessions() {
 							if(SiegeWarSettings.isNationSiegeImmunityEnabled())
 								siege.recordPrimaryTownGovernment();
 
-							//Continue to next siege if there were no battle points
-							if(siege.getAttackerBattlePoints() == 0 && siege.getDefenderBattlePoints() == 0) {
-								if(SiegeWarSettings.isNationSiegeImmunityEnabled())
-									SiegeController.saveSiege(siege);
-								continue;
-							}
-
-							//Calculate result
-							int battlePointsOfWinner;
-							if (siege.getAttackerBattlePoints() > siege.getDefenderBattlePoints()) {
-								battlePointsOfWinner = siege.getAttackerBattlePoints();
-							} else if (siege.getAttackerBattlePoints() < siege.getDefenderBattlePoints()) {
-								battlePointsOfWinner = -siege.getDefenderBattlePoints();
-							} else {
-								battlePointsOfWinner = 0;
+							//If any battle points were gained, calculate a result
+							if(siege.getAttackerBattlePoints() > 0 && siege.getDefenderBattlePoints() > 0) {
```

Comment:
This should probably be an || instead of an && no?

Reply:
Yes thanks, will fix

Reply:
Fixed